arch/ua_network_tcp.c

```diff
@@ -101,7 +101,9 @@ connection_recv(UA_Connection *connection, UA_ByteString *response,
         if(resultsize == -1) {
             /* The call to select was interrupted manually. Act as if it timed
              * out */
+#ifndef UNDER_CE /* What shall we do here? */
             if(errno == EINTR)
```

Comment:
This should be `UA_ERRNO` instead of `errno`. I will change that in another PR (#2042). Then you can change `UA_ERRNO` in your architecture definition and remove this change (rebase after the other PR is merged)